app/models/manageiq/providers/openstack/cloud_manager/flavor.rb

```diff
@@ -3,7 +3,12 @@ class ManageIQ::Providers::Openstack::CloudManager::Flavor < ::Flavor
 
   def self.raw_create_flavor(ext_management_system, create_options)
     ext_management_system.with_provider_connection({:service => 'Compute'}) do |service|
-      service.flavors.create(create_options)
+      flavor = service.flavors.create(create_options)
```

Comment:
Although Fog shouldn't care about `cloud_tenant_refs`being passed to it as part of the options hash, I think it would still be good practice to remove it from the hash before passing it to `create`. It's not a valid parameter for that method, and we have no guarantees that the implementation of `create` in Fog won't change in such a way that causes it to break later.